less/home-page.less

```diff
@@ -25,6 +30,24 @@
       border-radius: 20px;
       height: 100%;
     }
+
+    .sky {
+      @media @phone {
+        display: none;
```

Comment:
Good job targeting individual pictures here by their class names. Like we talked about. You could also use &:not(:first-child){display: none;} here